### PR TITLE
bold button resolver

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts [hunk 1]
```diff
@@ -1,16 +1,62 @@
...
 
-    expect(isTextInSelection.resolve(mockContentWhenTextIsInSelection)).toBeTruthy();
+      // @ts-ignore
+      expect(isTextInSelection.resolve(mockContentWhenTextIsInSelection)).toBeTruthy();
 
-    const mockContentWhenTextIsNotInSelection = [{ type: { name: 'image' } }];
-    expect(isTextInSelection.resolve(mockContentWhenTextIsNotInSelection)).toBeFalsy();
+      const mockContentWhenTextIsNotInSelection = [{ type: { name: 'image' } }];
+      // @ts-ignore
+      expect(isTextInSelection.resolve(mockContentWhenTextIsNotInSelection)).toBeFalsy();
```
Comment (justame) [inline packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts:29]: use:
toBe(true)
toBe(false)

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts [hunk 2]
```diff
@@ -1,16 +1,62 @@
...
+      // @ts-ignore
+      const mockContent: TiptapNode[] = [{ type: 'paragraph' }];
+      expect(alwaysVisibleResolver.resolve(mockContent)).toBe(true);
+    });
   });
 
-  it('test isTextInSelection resolver - should check if the selection contains text', () => {
-    const mockContentWhenTextIsInSelection = [{ type: { name: 'text' } }];
+  describe('isTextInSelection', () => {
+    it('test isTextInSelection resolver - should check if the selection contains text', () => {
```
Comment (justame) [inline packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts:21]: remove "test isTextInSelection resolver"

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts [hunk 3]
```diff
@@ -1,16 +1,62 @@
...
+    });
+  });
+
+  describe('isTextContainsBold', () => {
+    it('should return true if selected text is bold', () => {
+      const mockContentWithBold: TiptapNode[] = [
+        // @ts-ignore
+        { type: { name: 'text' }, marks: [{ type: { name: 'bold' } }], text: 'World' },
+      ];
+      expect(isTextContainsBoldResolver.resolve(mockContentWithBold)).toBe(true);
```
Comment (justame) [inline packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts:39]: add negative test

**Comment by justame** [comment]
Looks good and clean. thank you.

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts (hunk 1 above)
Comment (noamtwix) [reply packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts]: fixed

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts (hunk 2 above)
Comment (noamtwix) [reply packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts]: fixed

File: packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts (hunk 3 above)
Comment (noamtwix) [reply packages/toolbars-v2/web/src/resolvers/tiptapResolvers.spec.ts:26]: fixed

**Pull request merged**